client/src/containers/CampaignPage.js

```diff
@@ -146,9 +212,19 @@ class CampaignPage extends Component {
             </Row>
           </Col>
           <Col md={3} xs={12} className="side-bar">
-            <div>
-              <SignCampaign signatureObj={this.props.signature} />
-              <SignatureList signatures={this.props.signature.signatures} />
+            <SignCampaign signatureObj={this.props.signature} />
+            <div className="text-center">
+              <CollapsePanel
+                signatures={this.props.signature.signatures}
+                titleText="See Who's Signed"
+                body={
+                  <ol>
```

Comment:
SignatureList contains a single ul. wrapping it in a ol/li elements is redundant

Reply:
Solved. nice catch![bitmoji](https://render.bitstrips.com/v2/cpanel/fbf70eb2-c012-465e-9c1e-7b092675dfd8-9e17bbb5-ab70-4793-bdf9-b6ef51f2a3a3-v1.png?transparent=1&palette=1&width=246)